commit d8732a59d3b1f2d0bebf43f294df7e9f333abde4
Author: Frank Reininghaus <frank78ac@googlemail.com>
Date:   Thu Apr 5 18:59:58 2012 +0200

    Reduce the timeout in KItemListKeyboardSearchManager to 1 second
    
    It turned out that the longer timeout, introduced recently in
    02eab49b2de51c31fe46a0d9501327b579b3648e, not only made multi-letter
    keyboard searches easier, but also had some unwanted side effects. I
    hope that 1 second, which is between the previous value of 5 seconds and
    the pre-KDE 4.8.2 value of 0.4 seconds, is a compromise which will fit
    most users' needs.
    We will try to improve the situation further in future releases by
    providing visual feedback about the keyboard search, but such a change
    would not be suitable for a bug-fix release.
    CCBUG:297458
    CCBUG:297488

diff --git a/dolphin/src/kitemviews/kitemlistkeyboardsearchmanager.cpp b/dolphin/src/kitemviews/kitemlistkeyboardsearchmanager.cpp
index f4dc1a5..592605a 100644
--- a/dolphin/src/kitemviews/kitemlistkeyboardsearchmanager.cpp
+++ b/dolphin/src/kitemviews/kitemlistkeyboardsearchmanager.cpp
@@ -29,7 +29,7 @@
 
 KItemListKeyboardSearchManager::KItemListKeyboardSearchManager(QObject* parent) :
     QObject(parent),
-    m_timeout(5000)
+    m_timeout(1000)
 {
     m_keyboardInputTime.invalidate();
 }
